### DISCUSSION TITLE
Zoom Group, Practical 3

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#439] [Old]
To post your links for practical series 3.

**Comment by ISchwandner** [comment]
Hi everyone,

Here is the link to my repository for practical 3. 
https://github.com/IBAHCM/ISchwandnerSeries03.git

Would be very grateful if some of you could check whether you can
(1) install my package
(2) see what demos are available
(3) run the demos
and 
(4) compile reports from the demos.

Thank you very much!

Best,
Imo

**Comment by sreedevi-gummuluri** [comment]
Hi everyone,
Here's a link to my repository for practical 3 : https://github.com/IBAHCM/SreedeviGummuluriSeries03.git

I'd appreciate any feedback and checks to make sure it works for other.

Thanks,
Sreedevi

**Comment by maryadejoh** [comment]
Hi everyone.
Here is a link to my repository. Still trying to bring it up to par at the moment. 
I will update as soon as this is done. https://github.com/IBAHCM/maryadejohSeries03.git

Regards,
Mary

**Reply by ISchwandner** [reply]
Hi Mary,

I think you still need to give us access to it :)

Imo

**Reply by maryadejoh** [reply]
Swoosh!
I'll do that now
Thanks for noting that.

**Reply by ISchwandner** [reply]
Hi Mary, I still can't access it for some reason. getting that 404, "this is not the webpage you are looking for" error still :/
